modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/converter/ObjectDefinitionDTOConverter.java

```diff
@@ -0,0 +1,123 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.admin.rest.internal.dto.v1_0.converter;
+
+import com.liferay.notification.service.NotificationTemplateLocalService;
+import com.liferay.object.admin.rest.dto.v1_0.ObjectDefinition;
+import com.liferay.object.admin.rest.dto.v1_0.ObjectField;
+import com.liferay.object.admin.rest.dto.v1_0.ObjectValidationRule;
+import com.liferay.object.admin.rest.dto.v1_0.ObjectView;
+import com.liferay.object.admin.rest.internal.dto.v1_0.converter.constants.DTOConverterConstants;
+import com.liferay.object.admin.rest.internal.dto.v1_0.util.ObjectDefinitionUtil;
+import com.liferay.object.model.ObjectRelationship;
+import com.liferay.object.service.ObjectActionLocalService;
+import com.liferay.object.service.ObjectDefinitionLocalService;
+import com.liferay.object.service.ObjectFieldLocalService;
+import com.liferay.object.service.ObjectLayoutLocalService;
+import com.liferay.object.service.ObjectRelationshipLocalService;
+import com.liferay.object.service.ObjectValidationRuleLocalService;
+import com.liferay.object.service.ObjectViewLocalService;
+import com.liferay.object.system.SystemObjectDefinitionManagerRegistry;
+import com.liferay.portal.vulcan.dto.converter.DTOConverter;
+import com.liferay.portal.vulcan.dto.converter.DTOConverterContext;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Feliphe Marinho
+ */
+@Component(
+	property = "dto.class.name=com.liferay.object.model.ObjectDefinition",
+	service = DTOConverter.class
+)
+public class ObjectDefinitionDTOConverter
+	implements DTOConverter
+		<com.liferay.object.model.ObjectDefinition, ObjectDefinition> {
+
+	@Override
+	public String getContentType() {
+		return ObjectDefinition.class.getSimpleName();
+	}
+
+	@Override
+	public ObjectDefinition toDTO(
+			DTOConverterContext dtoConverterContext,
+			com.liferay.object.model.ObjectDefinition objectDefinition)
+		throws Exception {
+
+		if (objectDefinition == null) {
+			return null;
+		}
+
+		ObjectDefinition objectDefinition1 =
```

Comment:
Not sure if this is naming is matching the pattern as there is no objectDefinition2. What about something like `dtoObjectDefinition`?